source/install/install-rhel-71-postgresql.rst

```diff
@@ -7,7 +7,7 @@ Installing PostgreSQL Database
 
 2. Download the PostgreSQL 9.4 Yum repository.
 
-  ``wget https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm``
+  ``curl -O https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm``
```

Comment:
Sorry, one last thing (I hope).

It looks like PostgreSQL provides a separate rpm for each distro that these instructions apply to https://yum.postgresql.org/repopackages.php

```
https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm
https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-3.noarch.rpm
https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-sl94-9.4-3.noarch.rpm
https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-oraclelinux94-9.4-3.noarch.rpm
```
The format for .rst should look like this:

```
For RHEL 7
    curl -O https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm
For CentOS 7
    curl -O https://download.postgresql.org/pub/repos/yum/9.4/redhat/centos-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm

etc
```
Can you add that in?